batch/sql/11708-is-job-cancelled.sql

```diff
@@ -0,0 +1,382 @@
+DELIMITER $$
```

Comment:
Compare with https://github.com/hail-is/hail/blob/62efb32d9544b0c964019345281e89b235f02f92/batch/sql/finalize-job-groups.sql#L4